WEB/src/views/ProfileView.vue

```diff
@@ -0,0 +1,93 @@
+<script setup lang="ts">
+  import Profile from '@/components/profile/Profile.vue'
+  import Class from '@/components/profile/Class.vue'
+  import Success from '@/components/profile/Success.vue'
+  import MainHeader from '@/components/headers/MainHeader.vue'
+  import { ref } from 'vue'
+  //const store = useCardStore()
+  //const remaining = computed(() => store.remaining)
+  const comp = ref(0)
+
+  function changeComp(nb: number){
+    comp.value = nb;
+  }
+
+  function logOut(){
+    //à completer
+  }
+</script>
+
+<template>
+  <div>
+    <MainHeader />
+  </div>
+  <div class="contents">
+    <div class="buttons">
+      <button @click="changeComp(0)">
+        Profil
+      </button>
+      <button @click="changeComp(1)">
+        Classe
+      </button>
+      <button @click="changeComp(2)">
+        Succès
+      </button>
+      <button @click="logOut">
+        Log out
+      </button>
+    </div>
+    <div class="compP">
+      <div v-if="comp == 0">
+        <Profile />
+      </div>
+      <div v-else-if="comp == 1">
+        <Class />
+      </div>
+      <div v-else="comp == 2">
```

Comment:
That's fine, it works but for a performance issue, it would be more efficient to use the "v-show" directive instead of "v-if" because a "v-show" simply does a "display: none;", whereas a "v-if, v-else" re-renders the component!